docs/introduction.mdx

```diff
@@ -77,7 +77,7 @@ import Stack from "@mui/material/Stack";
       </div>
       <div>
         <div class="card intro-card">
-          <a href="/docs/device-iq">
+          <a href="/docs/device360">
```

Comment:
We should change all instances of Device IQ in this file to Device 360. And also add some sort of sub heading that makes sense:
<img width="367" alt="image" src="https://github.com/gobeyondidentity/customer-docs/assets/6578679/62184dd7-fcba-466b-ba29-ec4ead7fb428">